src/main/java/hu/blackbelt/structured/map/proxy/CompositeClassLoader.java

```diff
@@ -1,5 +1,25 @@
 package hu.blackbelt.structured.map.proxy;
 
+/*-
+ * #%L
+ * Structured map proxy
+ * %%
+ * Copyright (C) 2018 - 2022 BlackBelt Technology
```

Comment:
Is it possible to set the end date from a variable? It would eliminate the need for us to manually bump it every year.